unified-runtime/source/loader/layers/tracing/ur_tracing_layer.cpp

```diff
@@ -67,16 +78,23 @@ uint64_t context_t::notify_begin(uint32_t id, const char *name, void *args) {
     return UINT64_MAX;
   }
 
-  if (auto loc = codelocData.get_codeloc()) {
-    xpti::payload_t payload =
-        xpti::payload_t(loc->functionName, loc->sourceFile, loc->lineNumber,
-                        loc->columnNumber, nullptr);
-    uint64_t InstanceNumber{};
```

Comment:
Given that this is removing code location info from the tracing layer, we also need to remove the corresponding UR API and SYCL integration:
https://github.com/intel/llvm/blob/0433e4d6f5c97f5870d4ffabcb3a7779ef9cf596/sycl/source/detail/ur.cpp#L191-L192

https://github.com/intel/llvm/blob/0433e4d6f5c97f5870d4ffabcb3a7779ef9cf596/sycl/source/detail/common.cpp#L40
Here's the UR API:
https://github.com/intel/llvm/blob/sycl/unified-runtime/include/ur_api.h#L1174

https://github.com/intel/llvm/blob/sycl/unified-runtime/test/layers/tracing/codeloc.cpp